src/System.Private.CoreLib/src/System/Reflection/LoadNativeLibraryArgs.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace System.Reflection
```

Comment:
Where do you think `LoadNativeLibraryArgs` should be placed? Is `System.Reflection` a good place for it?

Reply:
All public APIs related to this feature should live in  System.Runtime.InteropServices.